cli/package.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
Looks like copy pasted file

Reply:
@volovyks You are right, didn't see that this happened.